### PR TITLE
Add LANCOM devices

### PR DESCRIPTION
**Pull request by mag-marc-schulz**
Hello,
I've added some LANCOM Devices. AccesPoint, Router and LTE-Extender.

Thanks in advance and kind regards,
Marc

### CODE REVIEW

File: device-types/LANCOM/1793VA-4G.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: LANCOM
+model: 1793VA-4G
+slug: lancom-1793va-4g
+part_number: '62116'
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/LANCOM/1793VA-4G.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/LANCOM/730-4G-plus.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: LANCOM
+model: 730-4G+
+slug: lancom-730-4g-plus
+part_number: '61705'
+u_height: 1
+is_full_depth: false
```
Comment (danner26) [inline device-types/LANCOM/730-4G-plus.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/LANCOM/LN-830E.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+manufacturer: LANCOM
+model: LN-830E
+slug: lancom-ln-830e
+part_number: '61747'
+u_height: 1
+is_full_depth: false
+airflow: passive
+comments: '[LANCOM LN-830E Data sheet](https://my.lancom-systems.de/download/documentation/hardware/?id=7d727fda5f49452344fa706eb9188d68&file=/DS_LN-830E_DE.pdf)'
```
Comment (danner26) [inline device-types/LANCOM/LN-830E.yaml:9]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/LANCOM/1793VA-4G.yaml (hunk 1 above)
Comment (mag-marc-schulz) [reply device-types/LANCOM/1793VA-4G.yaml:7]: Hi,
I added the weight.

For the airflow i only found the hint "1 silent fan". I didn't found any directions. From the pictures I assume it is right to left oder left to right. But that's just a guess.
Unfortunately, I don't have physical access to the device either.
How should we proceed here?

File: device-types/LANCOM/730-4G-plus.yaml (hunk 2 above)
Comment (mag-marc-schulz) [reply device-types/LANCOM/730-4G-plus.yaml:7]: I added airflow and weight

File: device-types/LANCOM/LN-830E.yaml (hunk 3 above)
Comment (mag-marc-schulz) [reply device-types/LANCOM/LN-830E.yaml:11]: I added the weight.

**Review by danner26 — APPROVED** [review]

**Pull request merged**